src/components/AppSpinner.tsx

```diff
@@ -1,11 +1,26 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import NavFrontendSpinner from 'nav-frontend-spinner';
+import styled from 'styled-components';
 
-const AppSpinner = () => {
-  return (
-    <NavFrontendSpinner type="XL">
-      Vent litt mens siden laster
-    </NavFrontendSpinner>
+interface Props {
+  laster: boolean;
+  children: ReactElement;
+}
+
+const SpinnerContainer = styled.div`
+  text-align: center;
+  margin-top: 64px;
```

Comment:
Vi pleier å bruke relativ størrelse(em) og ikke pixler for margin/padding.

Reply:
Endrer